.travis.yml

```diff
@@ -59,16 +59,16 @@ before_install:
 
 install:
     - build_wheel $REPO_DIR $PLAT
-    # Additionally build sdist on tag and not bundling.
-    - if [ -z $"FREETYPEPY_BUNDLE_FT" && -n "$TRAVIS_TAG" ]; then python setup.py sdist; fi
 
 script:
     - install_run $PLAT
 
 after_success:
-  # copy compiled wheels to dist/ where Travis `dpl` tool can find them and 
+  # Copy compiled wheels to dist/ where Travis `dpl` tool can find them and 
   # upload to PyPI
   - if [ -n "$TRAVIS_TAG" ]; then mkdir -p dist; cp wheelhouse/*.whl dist; fi
+  # Additionally build sdist on tag and not bundling.
+  - if [ -z $"FREETYPEPY_BUNDLE_FT" && -n "$TRAVIS_TAG" ]; then python setup.py sdist; fi
```

Comment:
again this is not valid. You need to move the `&&` outside the brackets, like so:

```sh
if [ -z "$FREETYPEPY_BUNDLE_FT" ] && [ -n "$TRAVIS_TAG" ]; then python setup.py sdist; fi
```

but you don't want to build the sdist for every build, only once. So export another variable and check that here

Reply:
Isn't that what's done here? It makes sure that 1. no bundling is going on 2. we are tagged?

Reply:
because you're assuming that there is only one build that does not export FREETYPEPY_BUNDLE_FT variable; it may not always be like that, e.g. say we add another build to do linting, etc.